docs/OrganizingCommittee/Committee-formation.md

```diff
@@ -0,0 +1,84 @@
+# Committee Formation
+
+Usually a hack week is spearheaded by one or several individual who have the motivation to host a hack week for a specific community. Once topic of a hackweek is formulated, the process of gathering together other committee memebers should begin, ideally 8 - 12 months prior to the event. 

```

Comment:
* ` [...] several individual*s* [...]` ?
* I would argue that gathering the committee members should happen at the *latest* 12 months in advance, especially for a new hack week. 8 months for a newly developed hack week might be a bit stressful. :)